README.md

```diff
@@ -7,3 +7,19 @@ chemtrail-distribution system based on flight paths supplied by the user and the
 to carry that payload alongside regular passengers. Results should be in the form of square meters covered and
 energy cost in Joules, as well as an optional visual representation on a map and the density of the gas at ground level. 
 The threshold gas level for visualization on the ground, as well as the plane height, should be specifiable by the user.
+
+Upon starting the game you will see a welcome screen:
+
+![welcome screen](mustards_game/screenshots/welcome_screen.png)
+
+Start game screen:
+
+![start game screen](mustards_game/screenshots/start_screen.png)
```

Comment:
do not think we need it
let's add another screenshot of the game itself, where more area is covered

Reply:
more area is covered by the gas you mean?

Reply:
yes